src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/ProvenanceSpec.scala

```diff
@@ -234,26 +233,66 @@ class ProvenanceSpec extends AnyFlatSpec with FileSystemSupport with XmlSupport
     val ddmOut = testConfig("archaeology").ddmTransformer.transform(ddmIn, "easy-dataset:123")
       .getOrElse(fail("no DDM returned"))
 
-    Provenance.compare(
-      (ddmIn \ "dcmiMetadata").head,
-      (ddmOut \ "dcmiMetadata").head,
-      "http://easy.dans.knaw.nl/schemas/md/ddm/"
-    ) shouldBe None
+    Provenance.compareDDM(ddmIn, ddmOut) shouldBe None
 
     assume(schemaIsAvailable)
     validate(ddmIn) shouldBe Success(())
   }
+  it should "change namespace" in {
+    val ddmIn = {
+      <ddm:DDM xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation=" http://easy.dans.knaw.nl/schemas/md/ddm/ https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd">
+        <ddm:profile>
+          <dc:title>blabla</dc:title>
+        </ddm:profile>
+      </ddm:DDM>
+    }
+
+    val expectedDdm = {
+      <ddm:DDM xmlns:ddm="http://easy.dans.knaw.nl/dataset/ddm-v2/"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation=" http://easy.dans.knaw.nl/schemas/md/ddm/ https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd">
+        <ddm:profile>
+          <dc:title>blabla</dc:title>
+          <ddm:personalData present="Yes"/>
+        </ddm:profile>
+      </ddm:DDM>
+    }
+
+    val expectedProv = {
+      <prov:provenance xsi:schemaLocation="http://easy.dans.knaw.nl/schemas/bag/metadata/prov/ https://easy.dans.knaw.nl/schemas/bag/metadata/prov/provenance.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:prov="http://easy.dans.knaw.nl/schemas/bag/metadata/prov/">
+        <prov:migration app="EasyConvertBagToDepositApp" version="1.0.5" date="2020-02-02">
+        <prov:file scheme="http://easy.dans.knaw.nl/schemas/md/ddm/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/">
+        <prov:old/>
+        <prov:new><ddm:personalData present="Yes"/></prov:new>
+      </prov:file>
+        </prov:migration>
+      </prov:provenance>
+    }
+
+    // a few steps of EasyConvertBagToDepositApp.addProps
+    val ddmOut = testConfig("archaeology").copy(ddmVersion = V2).ddmTransformer
+      .transform(ddmIn, "easy-dataset:123", containsPrivacySensitiveData = "true")
+      .getOrElse(fail("no DDM returned"))
+    val actualProv = provenanceBuilder.collectChangesInXmls(List(Provenance.compareDDM(ddmIn, ddmOut)))
+
+    normalized(ddmOut) shouldBe normalized(expectedDdm)
+    normalized(actualProv) shouldBe normalized(expectedProv)
+
+    // check what was stripped by normalize to avoid random order
+    ddmOut.serialize should include("""xmlns:ddm="http://schemas.dans.knaw.nl/dataset/ddm-v2/"""")
+
+    assume(schemaIsAvailable)
+    validate(actualProv) shouldBe a[Success[_]]
```

Comment:
Results (of course) in 

    Invalid content was found starting with element 'ddm:personalData'.